db_helper.php

```diff
@@ -1,98 +1,129 @@
 <?php
+if (!file_exists('config.php')) {
+    exit('Please copy \'config.sample.php\' to \'config.php\'');
+}
+require_once 'config.php';
 
+if (Config::DEBUG) {
```

Comment:
Oh, I think it is important to always have the error reporting on, otherwise the exception handling will not work. At least this is what *the internet* told me. Besides, I don't thing debug dispatching is needed here.

Reply:
Okay, for mysql errors this might be the case. But then the PHP error should be turned off (in production).

https://secure.php.net/manual/en/mysqli-driver.report-mode.php

Reply:
If you know how to do this without affecting the exception handling for db operations, feel free to do so.

Reply:
PHP error reporting should be disabled in the php.ini of the server.

Reply:
Well, that's clearly a deployment thing and out of scope of this project.